src/pybind/mgr/dashboard/frontend/src/app/shared/components/motd/motd.component.scss

```diff
@@ -0,0 +1,4 @@
+::ng-deep .alert {
```

Comment:
This selector is deprecated and we should avoid using it. See:
https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

Moreover, if you use it without `:host` the style can leak into other components that lead to unexpected issues.

Reply:
@alfonsomthd I already raised this concern in another PR a while ago, so it seems that we still need "ng-deep" in some places... But I'd be very cautious when adding it, if there's an alternative solution.

Reply:
@epuertat The point is what we want to achieve:
1) If we want to apply the style to the component and its children: then go for `:host ::ng-deep`.
2) If the style needs to be applied at a global level, then move it to the global style file(s) and get rid of `::ng-deep` here as it is harder to find out where that style comes from when debugging.

In any case, `::ng-deep` not preceded by `:host`is discouraged.